components/vc/pdu/include/HW/HW_gpio_componentSpecific.h

```diff
@@ -9,12 +9,87 @@
  *                             T Y P E D E F S
  ******************************************************************************/
 
-typedef enum
+
+typedef enum 
 {
-    HW_GPIO_CAN1_RX = 0U,
-    HW_GPIO_CAN1_TX,
+    HW_GPIO_BMS2_EN = 0U,
+    HW_GPIO_ACCUM_EN,
+    HW_GPIO_LP2_LATCH,
+    HW_GPIO_BSPD_MEM,
+    HW_GPIO_SPI_CS_SD, /*CS ELEMENT*/
+    HW_GPIO_VCU3_EN,
+    HW_GPIO_MC_EN,
+    HW_GPIO_LP5_LATCH,
+    /*----ADC/DAC STARTS HERE----*/
+    HW_GPIO_ADC_LP2_SNS, 
+    HW_GPIO_ADC_LP3_SNS,
+    HW_GPIO_ADC_LP4_SNS,
+    HW_GPIO_ADC_LP5_SNS,

```

Comment:
Rename to `HW_GPIO_ADC_MUX_LP*_SNS` So that its clearly a mux and not the same as HP and consistent with the MUX_LP_SEL* lines